nixos/modules/services/hardware/openrgb.nix

```diff
@@ -34,13 +34,13 @@ in {
     services.udev.packages = [ cfg.package ];
 
     boot.kernelModules = [ "i2c-dev" ]
-     ++ lib.optionals (cfg.motherboard == "amd") [ "i2c-piix" ]
+     ++ lib.optionals (cfg.motherboard == "amd") [ "i2c-piix4" ]
      ++ lib.optionals (cfg.motherboard == "intel") [ "i2c-i801" ];
 
     systemd.services.openrgb = {
       description = "OpenRGB server daemon";
       wantedBy = [ "multi-user.target" ];
-
+      environment.XDG_CONFIG_HOME = "/var/lib/";
```

Comment:
Just curious, does just setting `serviceConfig.WorkingDirectory = "/var/lib/OpenRGB"` work as well?

Reply:
it does, assuming the file exists so ive added a tmpfile rule as well.

Reply:
Systemd I believe should create it for you if you add `serviceConfig.StateDirectory = "OpenRGB"` too